src/projects/pull-request/get-comments.html

```diff
@@ -59,15 +59,15 @@
       is: 'get-comments',
       behaviors: [ProjectBehavior],
       properties: {
+        pullRequest: Object,
         commentsFbPath: {
           type: String,
-          computed: '_computeCommentsFbPath(project, number)'
+          computed: '_computeCommentsFbPath(pullRequest)'
```

Comment:
get-comments.html:93 Uncaught TypeError: Cannot read property 'owner' of undefined